extensions/jaxb/deployment/src/main/java/io/quarkus/jaxb/deployment/JaxbProcessor.java

```diff
@@ -178,6 +198,44 @@ void processAnnotationsAndIndexFiles(
         }
     }
 
+    private void produceProxyIfExist(BuildProducer<NativeImageProxyDefinitionBuildItem> proxies, String s) {
```

Comment:
Please avoid using one character variables that are meaningless. Here it's an `interfaceName`.